website/components/plugin-tier-label/style.module.css

```diff
@@ -40,11 +45,5 @@ to match Terraform Registry tier labels.
 
 .icon {
   margin-right: 4px;
-
-  & svg {
-    display: block;
-    & [fill] {
-      fill: var(--text-color);
-    }
-  }
+  color: var(--text-color);
```

Comment:
Have refactored the `PluginTierLabel` component slightly - have instead used `BadgeHeader`, which leverages `PluginTierLabel` internally.